docs/getting-started/installation.md

```diff
@@ -27,6 +27,37 @@ Chart.js built files are also available through [jsDelivr](https://www.jsdelivr.
 
 <https://www.jsdelivr.com/package/npm/chart.js?path=dist>
 
+## Security Considerations
+
+When loading Chart.js from a CDN, it's recommended to use **Subresource Integrity (SRI)** to protect against tampered or malicious scripts. Here are secure examples:
+
+### CDNJS with SRI
```

Comment:
Since this feature is not CDN specific and both CDN's are already described above I would keep this generic and just show a default config